Developing in home dir causes type issues

From [`tsconfig#typeRoots`](https://www.typescriptlang.org/tsconfig#typeRoots),
> By default all visible ”@types” packages are included in your compilation. Packages in node_modules/@types of any enclosing folder are considered visible. For example, that means packages within ./node_modules/@types/, ../node_modules/@types/, ../../node_modules/@types/, and so on.

This means for developers developing from somewhere inside their home dir, globally installed `@types` interfere and conflict with the types in `starknet-hardhat-plugin` directory. Resulting in errors like these,

`../../..` Points to my home dir.

```
../../../node_modules/@types/eslint/helpers.d.ts:1:6 - error TS2300: Duplicate identifier 'Prepend'.
'Prepend' was also declared in node_modules/@types/eslint/helpers.d.ts:1:6
```

<img width="1213" alt="image" src="https://user-images.githubusercontent.com/11048263/209186366-adc94537-2228-4a85-b2d9-29954250727f.png">

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -10,7 +10,8 @@
         "strictNullChecks": false,
         "declaration": true,
         "module": "commonjs",
-        "noImplicitOverride": true
+        "noImplicitOverride": true,
+        "typeRoots": ["./node_modules/@types"]
     },
     "include": ["./src", "config.json", "contract-artifacts/**/*.json"]
 }

